app/assemblers/assemble_notifications_list.rb

```diff
@@ -4,7 +4,7 @@ class AssembleNotificationsList
   initialize_with :user, :params
 
   def self.keys
-    %i[page per_page order]
+    %i[page per_page order for_header]
```

Comment:
The "for_header" bit below wasn't actually working properly as this param wasn't getting passed in.